contrib/google.golang.org/api/option.go

```diff
@@ -83,3 +85,11 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithEndpointMetadataEnabled allows to enable/disable the enriched Google endpoint metadata behavior for
```

Comment:
Let's add some details here on _why_ a customer would want to pass this option to override the default of `true` (totally ok to just copy some bits from the warning)

Reply:
I tried to write a shorter version of the warning here, please let me know if that works!